static/app/components/noProjectMessage.spec.tsx

```diff
@@ -96,4 +132,22 @@ describe('NoProjectMessage', function () {
       screen.getByText('You need at least one project to use this view')
     ).toBeInTheDocument();
   });
+
+  it('shows projects to superusers if membership is not required', function () {
```

Comment:
This is the test that would have failed on the previous PR

Reply:
Thanks for the test 👏